### PR TITLE
Nix pill 1: address feedback on #25

### PR DESCRIPTION
**Pull request by michaelpj**
As requested.

### CODE REVIEW

File: pills/01-why-you-should-give-it-try.xml [hunk 1]
```diff
@@ -126,25 +125,25 @@
...
        where the hash uniquely identifies the derivation (this isn't quite true,
        it's a little more complex), and the name is the name of
        the derivation.
      </para>
      <para>
        Let's take a bash derivation as an example:
-       <literal>/nix/store/s4zia7hhqkin1di0f187b79sa2srhv6k-bash-4.2-p45/</literal>.
-       This picks out a directory that contains <literal>bin/bash</literal>.
+       <filename>/nix/store/s4zia7hhqkin1di0f187b79sa2srhv6k-bash-4.2-p45/</filename>.
+       This is a path in the Nix store which points to a directory that contains <filename>bin/bash</filename>.
```
Comment (grahamc) [inline pills/01-why-you-should-give-it-try.xml]: It doesn't point do a directory, it _is_ a directory which  contains bin/bash.

**Comment by grahamc** [comment]
Looking good, just a minor request. I could even make that change if you'd rather :)

File: pills/01-why-you-should-give-it-try.xml (hunk 1 above)
Comment (michaelpj) [reply pills/01-why-you-should-give-it-try.xml]: Fixed.

**Comment by michaelpj** [comment]
Pushed the fix. I'm happy for you to push changes to my PRs in general :)

**Comment by grahamc** [comment]
Thank you! FWIW PRs typically end up on  NixOS.org within ~1.5hrs First the commit builds https://hydra.nixos.org/jobset/nix-pills/master#tabs-evaluations every hour, then every ~15min or so NixOS.org fetches updated builds.

**Pull request merged**